pkg/core/interop/context.go

```diff
@@ -207,6 +207,9 @@ func (c *ContractMD) AddMethod(md *MethodAndPrice, desc *manifest.Method) {
 	index := sort.Search(len(c.Manifest.ABI.Methods), func(i int) bool {
 		md := c.Manifest.ABI.Methods[i]
 		if md.Name != desc.Name {
+			if md.Name == "getCandidateVote" && desc.Name == "getCandidates" {
```

Comment:
Such a kludge... How about comparing `strings.ToLower()` values? This method is not on the hot path.